.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
```

Comment:
Do we really need a yml config just to run few bash commands on commit and push? You can follow the original documenation: https://git-scm.com/docs/githooks